bridge/elrond/client.go

```diff
@@ -220,13 +229,17 @@ func (c *client) ProposeTransfer(_ context.Context, batch *bridge.Batch) (string
 			BigInt(tx.Amount)
 	}
 
+	batchData, errMarshal := json.Marshal(batch)
+	if errMarshal != nil {
+		c.log.Warn("Elrond: error not critical while serializing transaction", "error", errMarshal)
+	}
+
 	gasLimit := c.gasMapConfig.ProposeTransferBase + uint64(len(batch.Transactions))*c.gasMapConfig.ProposeTransferForEach
 	hash, err := c.sendTransaction(builder, gasLimit)
-
 	if err == nil {
-		c.log.Info("Elrond: Proposed transfer for batch ", batch.Id, " with hash ", hash)
+		c.log.Info("Elrond: Proposed transfer for batch ", batch.Id, "with hash", hash, "batch data", string(batchData))
 	} else {
-		c.log.Error("Elrond: Propose transfer errored", "error", err.Error())
+		c.log.Error("Elrond: Propose transfer errored", "batch data", string(batchData), "error", err.Error())
```

Comment:
optional: `err` instead of `err.Error()`

Reply:
done